daml-lf/transaction/src/test/scala/com/digitalasset/daml/lf/value/ValueSpec.scala

```diff
@@ -107,6 +106,26 @@ class ValueSpec
 
   }
 
+  "AbsoluteContractID.V1.build" - {
+
+    "rejects too long suffix" in {
+
+      def suffix(size: Int) =
+        Bytes.fromByteArray(Array.iterate(0.toByte, size)(b => (b + 1).toByte))
+
+      val hash = crypto.Hash.hashPrivateKey("some hash")
+      import AbsoluteContractId.V1.build
+      build(hash, suffix(0)) shouldBe 'right
+      // ...
+      AbsoluteContractId.V1.build(hash, suffix(94)) shouldBe 'right
+      AbsoluteContractId.V1.build(hash, suffix(95)) shouldBe 'left
+      AbsoluteContractId.V1.build(hash, suffix(96)) shouldBe 'left
+      AbsoluteContractId.V1.build(hash, suffix(127)) shouldBe 'left
```

Comment:
Ha ha very funny, I elided the rest of them

```suggestion
      build(hash, suffix(94)) shouldBe 'right
      build(hash, suffix(95)) shouldBe 'left
      build(hash, suffix(96)) shouldBe 'left
      build(hash, suffix(127)) shouldBe 'left
```